src/compiler/types.ts

```diff
@@ -316,6 +316,7 @@ namespace ts {
         // Property assignments
         PropertyAssignment,
         ShorthandPropertyAssignment,
+        SpreadObjectLiteralAssignment,

```

Comment:
I thought about it. but i think it is may be more confused to do., @sandersn  is putting up PR for spread operator, will see how he handle it over there